### PR TITLE
Implement padding to property and markdown presenter

### LINKED ISSUE #6896: Add padding to UI
Adding padding to remove the hard alignment of content would improve the look of the UI and relates particularly to markupView and propertyView.

The CLEM wrapper presenter currently added borders internally to implement padding for property presenter and markdown presenter used widely through CLEM and make the UI look better, but this is only applicable to CLEM views and not general presenters such as the summary and clock presenter.
 
The completed shift to .Net Core means the margin property is nbow available to easily implement padding.
At present adding margins to Property and MarkDown presenters meets most requirements.

### PR DESCRIPTION
**Pull request by lie112**
Resolves #6896

### CODE REVIEW

File: ApsimNG/Views/CLEM/CLEMView.cs [hunk 1]
```diff
@@ -132,17 +132,17 @@ public void AddTabView(string tabName, object control)
...
-                {
-                    border = 10;
-                }
-
-                hbox.BorderWidth = border;
+                //uint border = 0;
+                //if (tabName == "Properties")
+                //{
+                //    border = 5;
+                //}
```
Comment (hol353) [inline ApsimNG/Views/CLEM/CLEMView.cs:139]: Please remove the commented code.

File: ApsimNG/Views/CLEM/ModelDetailsWrapperView.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ public class ModelDetailsWrapperView : ViewBase, IModelDetailsWrapperView
         private Label modelTypeLabel = null;
         private Label modelDescriptionLabel = null;
         private LinkButton modelHelpLinkLabel = null;
-        private Label modelVersionLabel = null;
+        //private Label modelVersionLabel = null;
         private Viewport bottomView = null;
```
Comment (hol353) [inline ApsimNG/Views/CLEM/ModelDetailsWrapperView.cs:33]: More commented code.

File: ApsimNG/Views/CLEM/ModelDetailsWrapperView.cs [hunk 3]
```diff
@@ -110,14 +93,13 @@ public ModelDetailsWrapperView(ViewBase owner) : base(owner)
...
-
+            modelDescriptionLabel.MarginBottom = 5;
 
+            labels.PackStart(modelDescriptionLabel, false, true, 0);
             vbox1.PackStart(labels, false, true, 0);
 
             ScrolledWindow scroll = new ScrolledWindow();
-            scroll.ShadowType = ShadowType.EtchedIn;
+            scroll.ShadowType = ShadowType.None; //ShadowType.EtchedIn;
             scroll.SetPolicy(PolicyType.Automatic, PolicyType.Automatic);
```
Comment (hol353) [inline ApsimNG/Views/CLEM/ModelDetailsWrapperView.cs:103]: commented code.

**Review by hol353 — CHANGES_REQUESTED** [review]

**Comment by lie112** [comment]
Too easy to miss cleaning up commented code during development.

**Review by hol353 — APPROVED** [review]

**Pull request merged**